framework/database/seeds/CVsTableSeeder.php

```diff
@@ -10,7 +10,7 @@ class CVsTableSeeder extends Seeder
      */
     public function run()
     {
-        $users = \App\User::retrieveUsers();
+        $users = \App\User::all();
         foreach($users as $user)
         {
             factory(App\CV::class)->create(['user_id' => $user->id]);
```

Comment:
L'avantage est de ne pas avoir à connaître/utiliser la clé étrangère.

Ou _Adding Relations To Models_ http://laravel.com/docs/5.1/testing#model-factories mais pas sûr que ça s'applique à votre problème ici.